Source/Tools/Flax.Build/Bindings/BindingsGenerator.Cpp.cs

```diff
@@ -166,7 +166,7 @@ public static string GenerateCppWrapperVariantToNative(BuildData buildData, Type
             if (typeInfo.Type == "String")
                 return $"(StringView){value}";
             if (typeInfo.IsPtr && typeInfo.IsConst && typeInfo.Type == "Char")
-                return $"((StringView){value}).GetText()";
+                return $"((StringView){value}).GetNonTerminatedText()"; // This is a bug, as we need a null-terminated strig here. Any idea how to fix it?
```

Comment:
I think it's not a problem as `Variant` if it's String type it contains null-terminator character so it's safe to access it. It will be null if string is empty.

Reply:
Added comments in Variant.h to document this property.